### PR TITLE
added token validation for namespace & namespace getting saved with file

### PR DESCRIPTION
**Pull request by anubhavitis**

### CODE REVIEW

File: src/infra/ucan.js [hunk 1]
```diff
@@ -5,80 +5,114 @@ const ucans = require('ucans');
...
+  } catch (error) {
+    console.error('Error verifying UCAN with namespace:', error);
+    return false;
+  }
+}
+
+async function contractAddressValidation(contractAddress, invokerAddress, token, chainId) {
+  let invokerDid = null;
+
+  try {
```
Comment (nadeem-fileverse) [inline src/infra/ucan.js:33]: why use multiple try catch here .. this can be done in a single try catch right ?

File: src/infra/ucan.js [hunk 2]
```diff
@@ -5,80 +5,114 @@ const ucans = require('ucans');
...
 
-let verify = (req, res, next) => {
-  req.requestId = uuidv4();
-  console.log('req.requestId: ', req.requestId);
-  let token = req.headers['authorization']; // Express headers are auto converted to lowercase
-  if (token && token.startsWith('Bearer ')) {
-    // Remove Bearer from string
-    token = token.slice(7, token.length);
+
+async function namespaceValidation(namespace, invokerAddress, token) {
```
Comment (nadeem-fileverse) [inline src/infra/ucan.js:9]: may be rename to `validateNamespace` same for all new new functions

Comment (nadeem-fileverse) [reply src/infra/ucan.js:9]: `validateContractAddress` and so on

**Review by nadeem-fileverse — CHANGES_REQUESTED** [review]
PTAL

File: src/infra/ucan.js (hunk 1 above)
Comment (anubhavitis) [reply src/infra/ucan.js:33]: added different try catch to handle different errors with more readable logs.

File: src/infra/ucan.js (hunk 2 above)
Comment (anubhavitis) [reply src/infra/ucan.js:9]: alright, doing it.

**Review by nadeem-fileverse — APPROVED** [review]
LGTM

**Pull request merged**